### PR TITLE
Subtle Alert Style

### PR DESCRIPTION
**Pull request by joeljfischer**
This feature adds a new style of alert: the "subtle" alert, which does not take over the entire screen. The current alerts are moved to a style called "prominent."

### CODE REVIEW

**Comment by joeljfischer** [comment]
@theresalech This is ready for review

File: proposals/nnnn-alert-style-subtle.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Java Suite / HMI / RPC]
+
+## Introduction
+This feature adds a new style of alert: the "subtle" alert, which does not take over the entire screen. The current alerts are moved to a style called "prominent."
+
+## Motivation
+Competitive systems in this space often have "notification" style alerts, and user expectations are often set by mobile devices, which have notification alerts that don't take over the screen. To meet user and developer expectations, we should provide both prominent (modal) and subtle (notification) style alerts.
+
+Places where subtle alerts may be useful is for messaging apps that notify the user of a new message, a VOIP app notifying the user of an incoming call, or a navigation app notifying the user of an upcoming accident. These should have the ability to be presenting outside of the current screen-takeover method.
```
Comment (theresalech) [inline proposals/nnnn-alert-style-subtle.md]:
```suggestion
Places where subtle alerts may be useful include messaging apps that notify the user of a new message, a VOIP app notifying the user of an incoming call, or a navigation app notifying the user of an upcoming accident. These should have the ability to be presented outside of the current screen-takeover method.
```

File: proposals/nnnn-alert-style-subtle.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+## Alternatives considered
+
+### Alternative 1: Separate RPC
+Using a separate RPC allows us to (1) control this alert style separately from `Alert` in policies, and (2) to provide a callback for when the notification is touched. This would permit better control for the app developers than simply opening the app.
+
+```xml
+<function name="SubtleAlert" functionID="SubtleAlertID" messagetype="request">
+    <description>Shows an alert which typically consists of text-to-speech message and text on the display. At least either alertText1, alertText2 or TTSChunks need to be provided.</description>
+    
+    <description>Shows an alert which typically consists of text-to-speech message and text on the display. At least either alertText1, alertText2 or TTSChunks need to be provided.</description>
```
Comment (theresalech) [inline proposals/nnnn-alert-style-subtle.md]: Is this description supposed to be here twice?

Comment (joeljfischer) [reply proposals/nnnn-alert-style-subtle.md]: 🤔 I don't know how that happened

**Comment by joeljfischer** [comment]
@theresalech review issues fixed!

**Pull request merged**